clients/java-logger/shell/src/main/java/com/abcxyz/lumberjack/loggingshell/LoggingShellApplication.java

```diff
@@ -16,13 +16,45 @@
 
 package com.abcxyz.lumberjack.loggingshell;
 
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 /** Entry point for the Logging Shell/Test app. */
 @SpringBootApplication
 public class LoggingShellApplication {
-  public static void main(String[] args) {
+  public static void main(String[] args) throws IOException {
+    HttpServer jwkServer = HttpServer.create(new InetSocketAddress(8081), 0);
+    jwkServer.createContext("/.well-known/jwks", new JWKHandler());
+    jwkServer.setExecutor(null); // creates a default executor
+    jwkServer.start();
     SpringApplication.run(LoggingShellApplication.class, args);
   }
+
+  static class JWKHandler implements HttpHandler {
```

Comment:
Should we make this as util as well?

Reply:
I found java http and java grpc has two pom.xml. We can make this as a util but we can do it in separate PR.

Reply:
This belongs to jvs testing logics, we can add it in jvs. Not a blocker, can be an improvement later on.

Reply:
Will create a separate issue for it.

Reply:
https://github.com/abcxyz/lumberjack/issues/391